blogs/whats_new_1.6/index.md

```diff
@@ -0,0 +1,225 @@
+
+### eBPF Agent metrics
+
+You can view statistics for eBPF Agent.  This is in Web console under **Observe > Dashboards**.  Select **NetObserv / Health** in the Dashboard dropdown (Figure 5).  There are graphs for eviction rates, dropped flow rates, and more.
+
+![eBPF Agent statistics](images/ebpf_agent_statistics.png)
+_<div style="text-align: center">Figure 5: eBPF Agent statistics</div>_
+
+### Other eBPF changes
+
+Here are a few other minor eBPF changes.
+
+1. Option to alert if it's dropping flows<br>
+There is a new option in FlowCollector to control whether an alert is shown if it's dropping flows (Figure 6).  By default, this is enabled.
+
+![Disable alerts](images/ebpf_metrics_configuration.png)
+_<div style="text-align: center">Figure 6: eBPF Agent configuration - Disable alerts</div>_
+
+2. Use enhanced TCx hook<br>
+Internally, the eBPF Agent uses the Traffic Control (TC) hook to probe the ingress and egress traffic on the interfaces.  In OCP 4.16, which upgrades RHEL to 9.4, it can leverage the enhanced TCx hook for performance gains.
+
+3. Always save DNS flags<br>
+If the *DNSTracking* feature is enabled, regardless of the sampling setting, it always saves the DNS id and flags.  Previously, it would discard this data if the flow didn't exist.
+
+
+## Flow Collector Enhancements
+
+A few enhancements were made in Flow Collector.  They are:
+
+1. Filter on not 'ip'<br>
+In **Observe > Network Traffic**, for the IP-related filter fields, such as SourceIP, Destination IP, Source Node IP, and Destination Node IP, you can filter by *not* matching or excluding that IP or range (Figure 7).
+
+![Not 'ip' filter](images/not_ip_filter.png)
+_<div style="text-align: center">Figure 7: Not 'ip' filter</div>_
+
+2. De-duplication of flows<br>
+    When a flow is a duplicate on another interface, instead of saving multiple copies of that flow data, only one is stored now.  It keeps track of the list of interfaces that refer to the flow data.  This greatly reduces the amount of storage.
+
+    The raw JSON fields have been renamed and pluralized to `Interfaces` and `IfDirections`, since the type changed to a list from a single value.  Note the use of square brackets for the lists below.
+
+    ```
+      "IfDirections": [
+        0
+      ],
+      "Interfaces": [
+        "br-ex"
+      ],
+    ```
+
+3. Subnet labels<br>
+    You can define a list of names for CIDRs (IP ranges and/or hosts).  When the CIDR matches, the name appears in the flow data as the `SrcSubnetLabel` or `DestSubnetLabel` field.
+
+    YAML Example:
+
+    ```
+      processor:
+        subnetLabels:
+          customLabels:
+          - cidrs:
+            - 10.129.0.73/32  # provide a name for this host
+            name: querier
+          openShiftAutoDetect: false
```

Comment:
As said in the bug you reported ( https://issues.redhat.com/browse/NETOBSERV-1734 ) I don't see why you need to disable the auto-detect. If it's because the CIDR that you defined is in the cluster subnets, hence overlapping with the the auto-detected ones, maybe you could use a different example, with a cluster-external workload?

Reply:
Changed to use an external cidr example.